### PR TITLE
no renku plugin entry_point

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

**Review by volodymyrss — CHANGES_REQUESTED** [review]

File: setup.py [hunk 1]
```diff
@@ -41,7 +41,6 @@
     python_requires='>=3.9,<3.12',
     packages=packages,
     entry_points={
-        "renku": ["name_of_plugin = renkugraphvis.plugin"],
```
[the remark this answers is not in the document]
Comment (burnout87) [reply setup.py:44]: this plugin only provides CLI commands and the jupyter serverproxy servers, so it's not needed

**Comment by burnout87** [comment]
Another interesting behavior I noticed, which makes this PR actually important.

Within a renku session, where both of our plugins are installed, I noticed that with running a `renku run papermill ...` command, the hooks of the `renku-aqs-annotation` plugin are not triggered, and therefore the annotation are not saved within the project knowledge graph.

But instead, by removing the `entry_point`, `name_of_plugin`, from the `renku-graph-vis` plugin, the hooks are triggered.

It is quite convoluted, but I will create a dedicated issue.

**Comment by volodymyrss** [comment]
> Another interesting behavior I noticed, which makes this PR actually important.
> 
> Within a renku session, where both of our plugins are installed, I noticed that with running a `renku run papermill ...` command, the hooks of the `renku-aqs-annotation` plugin are not triggered, and therefore the annotation are not saved within the project knowledge graph.
> 
> But instead, by removing the `entry_point`, `name_of_plugin`, from the `renku-graph-vis` plugin, the hooks are triggered.
> 
> It is quite convoluted, but I will create a dedicated issue.

Hm, is it not just that identically-named hooks override each other?

**Review by volodymyrss — APPROVED** [review]

**Comment by burnout87** [comment]
> > Another interesting behavior I noticed, which makes this PR actually important.
> > Within a renku session, where both of our plugins are installed, I noticed that with running a `renku run papermill ...` command, the hooks of the `renku-aqs-annotation` plugin are not triggered, and therefore the annotation are not saved within the project knowledge graph.
> > But instead, by removing the `entry_point`, `name_of_plugin`, from the `renku-graph-vis` plugin, the hooks are triggered.
> > It is quite convoluted, but I will create a dedicated issue.
> 
> Hm, is it not just that identically-named hooks override each other?

https://github.com/oda-hub/renku-graph-vis/issues/16

**Pull request merged**